src/chart-review/encounters/encounters.component.tsx

```diff
@@ -6,6 +6,7 @@ import ConditionsDetailedSummary from "../../widgets/conditions/conditions-detai
 import { AllergyOverviewLevelTwo } from "../../widgets/allergies/allergy-card-level-two.component";
```

Comment:
Please this file. We will be using an alternative approach via configurations to set this up. The chart-widget class is also changing. I'll provide further explanation on call.

Reply:
> Please this file. We will be using an alternative approach via configurations to set this up. The chart-widget class is also changing. I'll provide further explanation on call.

should I delete the file or the imports?

Reply:
please delete the file. we will be adding a configuration to the default-views.tsx to create this.